### DISCUSSION TITLE
Application got selected what todo next...

### DISCUSSION THREAD

**Discussion by akshat5302** [github/india#666] [Students]
My application got selected but I'm bit confused because my main stream is ml and robotics but there is no organization with that stuff should I have to wait untill all organization get into the program or I have to apply only into these organisations only

**Comment by KSSSenapati** [comment]
My application also got approved yesterday. @akshat5302 you can only apply to those organisations which are listed [here](https://externship.github.in/organization/list) and at max you can apply to 4 projects
